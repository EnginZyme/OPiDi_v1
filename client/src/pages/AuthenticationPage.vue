<template>
  <div class="sign_in_card">
    <SignInView />
  </div>
</template>

<script>
import SignInView from "../components/views/SignInView";

/**
 * A component that renders the authentication page.
 */
export default {
  name: "AuthenticationPage",
  components: {
    SignInView,
  },
};
</script>

<style scoped>
.sign_in_card {
  justify-self: center;
  align-self: center;
}
</style>
