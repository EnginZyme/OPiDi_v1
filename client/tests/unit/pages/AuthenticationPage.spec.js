import { shallowMount } from "@vue/test-utils"
import AuthenticationPage from "@/pages/AuthenticationPage"


describe("shallowMounted AuthenticationPage", () => {
    const wrapper = shallowMount(AuthenticationPage)

    it("renders correctly", () => {
        expect(wrapper.element).toMatchSnapshot();
    })
})
