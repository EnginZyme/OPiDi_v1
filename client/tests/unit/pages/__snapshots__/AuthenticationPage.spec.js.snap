// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallowMounted AuthenticationPage renders correctly 1`] = `
<div
  class="sign_in_card"
>
  <signinview-stub />
</div>
`;
